packages/vscode/data/skills/likec4-dsl/references/identifier-validity.md

```
@@ -0,0 +1,90 @@
+# Identifier Validity (LikeC4 DSL)
+
+## Valid identifier form
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)